AllyAbilities.php

```diff
@@ -352,6 +353,18 @@ function DestroyAlly($player, $index, $skipDestroy = false, $fromCombat = false,
   return $cardID;
 }
 
+function CurrentEffectsContainWhenDefeated($player) {
+  global $currentTurnEffects;
+  for($i=0;$i<count($currentTurnEffects); $i+=CurrentTurnEffectPieces()) {
+    switch($currentTurnEffects[$i]) {
+      case "1272825113":
```

Comment:
Could you please add a comment with the name of the card? It's really helpful for maintenance.

Reply:
thank you that is now udpated. also worked for me to remove the webp files from the Unimplemented folder for those two